views/beneficial-owner-gov.html

```diff
@@ -257,27 +292,31 @@ <h1 class="govuk-heading-xl govuk-!-margin-0">Tell us about the government or pu
           items: [
             {
               value: "shares",
-              text: "They hold more than 25% of shares in the overseas entity"
+              text: "They hold more than 25% of shares in the overseas entity",
+              checked: corpNatureOfControl === "shares"
             },
             {
               value: "voting",
-              text: "They hold more than 25% of the voting rights in the overseas entity"
+              text: "They hold more than 25% of the voting rights in the overseas entity",
+              checked: corpNatureOfControl === "voting"
             },
             {
               value: "appoint",
-              text: "They hold the right to appoint or remove a majority of the board of directors of the overseas entity"
+              text: "They hold the right to appoint or remove a majority of the board of directors of the overseas entity",
+              checked: corpNatureOfControl === "appoint"
             },
             {
               value: "influence",
-              text: "They have the right to exercise significant influence or control over the overseas entity"
+              text: "They have the right to exercise significant influence or control over the overseas entity",
+              checked: corpNatureOfControl === "influence"
             }
           ]
         }) }}
 
         {{ govukRadios({
           classes: "govuk-radios",
-          idPrefix: "ownerSanctions",
-          name: "ownerSanctions",
+          idPrefix: "onSanctionsList",
```

Comment:
As this is a yes-no maybe "isOnSanctionList"

Reply:
will do